support-frontend/assets/components/subscriptionCheckouts/address/postcodeLookup.js

```diff
@@ -7,8 +7,14 @@ export type PostcodeFinderResult = {|
   city?: string,
 |};
 
-const getAddressesForPostcode = (postcode: string): Promise<PostcodeFinderResult[]> =>
-  fetch(postcodeLookupUrl(postcode))
-    .then(res => res.json());
+const getAddressesForPostcode = (postcode: string): Promise<PostcodeFinderResult[]> => {
+
+  if (window.guardian.checkoutPostcodeLookup) {
```

Comment:
@rupertbates How would I use `getGlobals` convention here? I'm not cool enough in react/redux terms to work it out.

Reply:
Just `getGlobal('checkoutPostcodeLookup')` doesn't need any redux involvement